lib/mayaUsd/commands/layerEditorCommand.cpp

```diff
@@ -1091,8 +1109,11 @@ MStatus LayerEditorCommand::parseArgs(const MArgList& argList)
             _subCommands.push_back(std::move(cmd));
         }
         if (argParser.isFlagSet(kLockLayerFlag)) {
-            bool lockIt = true;
-            argParser.getFlagArgument(kLockLayerFlag, 0, lockIt);
+            int lockValue = 0;
+            // 0 = Unlocked
```

Comment:
I wished there was an enum so that we would not have these magic values in the code.